packages/client/hmi-client/src/components/model/model-parts/tera-model-part.vue

```diff
@@ -1,25 +1,29 @@
 <template>
 	<ul>
-		<li v-for="({ base, children, isParent }, index) in items" :key="index" class="model-part">
-			<template v-if="isParent && !isEmpty(parentEditingState)">
+		<li
+			v-for="({ base, children, isParent }, index) in filteredItems.slice(firstRow, firstRow + MAX_NUMBER_OF_ROWS)"
+			:key="index"
+			class="model-part"
+		>
```

Comment:
I am apprehensive when we have a dynamic list and we use `:key=index` - we have had run into reactive issues where things doesn't work because while the underlying element changed, the index did not.

Is it possible to key by something else? for example the parameter/state ids should be unique.

We can probably still keep `getEditingState(index)` stuff, but vue binding/cache is somewhat out of our control so we should be careful around that.